torch/testing/_internal/jit_metaprogramming_utils.py

```diff
@@ -461,12 +468,12 @@ def make_module(script):
         return module
     return script_module
 
-def check_alias_annotation(method_name, args, kwargs):
+def check_alias_annotation(method_name, args, kwargs, *, aten_name, func_type='method'):
     formals, tensors, actuals = get_script_args(args)
-    call = get_call(method_name, 'method', actuals, kwargs)
+    call = get_call(method_name, func_type, actuals, kwargs)
```

Comment:
These operators don't have a method variant so this didn't work. But since this is called in the variants loop, it should really be testing the actual.